_includes/footer.html

```diff
@@ -0,0 +1,39 @@
+                        <p><a href="https://github.com/vacp2p" target="_blank">Check out the code</a></p>
+                        <p>It all lives in Github. Here, test it out, XYZ</p>
+                    </li>
+                    <li>
+                        <a href="https://forum.vac.dev/" target="_blank"><svg width="60" height="59" viewBox="0 0 60 59" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M31.0942 58.2734C23.9642 58.2734 17.4449 55.6679 12.4045 51.3578L0 53.0525L4.79223 41.2247C3.07059 37.4276 2.18189 33.3059 2.1859 29.1367C2.1859 13.0459 15.1286 0 31.0942 0C47.0586 0 60 13.0459 60 29.1367C60 45.2305 47.059 58.2734 31.0942 58.2734ZM46.8786 29.0534V28.9722C46.8786 20.5645 40.9494 14.5692 30.726 14.5692H19.6843V43.7059H30.5624C40.8652 43.7063 46.8786 37.4607 46.8786 29.0534ZM30.8478 36.5473H27.6181V21.7299H30.8478C35.5912 21.7299 38.7397 24.4342 38.7397 29.0983V29.1794C38.7397 33.8827 35.5912 36.5473 30.8478 36.5473Z" fill="#121212"/></svg></a>
+                        <p><a href="https://forum.vac.dev/" target="_blank">Join the research forum</a></p>
+                        <p>Find answers, pose questions, and join the open forum on Disqus</p>
```

Comment:
Replace with

Ask questions, find answers and discuss recent research at our form

Reply:
Done